StrangeCalculatorTask/src/main/java/Main.java

```diff
@@ -0,0 +1,13 @@
+import java.io.IOException;
+
+public class Main {
+    public static void main(String[] args) throws IOException {
+        ConsoleReader consoleReader = new ConsoleReader();
```

Comment:
Good practice to initialize object with type ReaderInterface, and then assign to it ConsoleReader. So if you will decide to change the type of reader you could do it easily.